go/client/kvstore_api_doc.go

```diff
@@ -2,12 +2,14 @@ package client
 
 const kvStoreAPIDoc = `"keybase kvstore api" provides a JSON API to fast, encrypted key-value storage. The "entryKey" and "namespace" fields are visible to the Keybase servers, "entryValue" is encrypted with the per-team-key. This is still a bit of a work in progress.
 
+If team is not included, then the command defaults to using the logged-in user's implicit self-team (only your user can see and decrypt this).
+
 EXAMPLES:
 
 Get an entry (always returns the latest revision, non-existent entries have a revision of 0):
 	{"method": "get", "params": {"options": {"team": "phoenix", "namespace": "pw-manager", "entryKey": "geocities"}}}
 
-Put an entry in your implicit self-team (only your user can see and decrypt this):
+Put an entry in your implicit self-team; either don't include team field, or if you want to write out the team name:
 	{"method": "put", "params": {"options": {"team": "yourusername,yourusername", "namespace": "pw-manager", "entryKey": "geocities", "entryValue": "all my secrets"}}}
```

Comment:
how about:
```
Put an entry:
{"method": "put", "params": {"options": {"namespace": "pw-manager", "entryKey": "geocities", "entryValue": "all my secrets"}}}
```
it's implicit from the later ones and the earlier explanation that this is encrypted just for you.